marklogic-data-hub/src/main/java/com/marklogic/hub/step/StepStatusListener.java

```diff
@@ -15,12 +15,14 @@
  */
 package com.marklogic.hub.step;
 
+import com.marklogic.hub.job.JobStatus;
+
 public interface StepStatusListener {
     /**
      *
      * @param jobId - the id of the job to change
      * @param percentComplete - the percentage of completeness expressed as an int
      * @param message - the message you'd like to send along with it
      */
-    void onStatusChange(String jobId, int percentComplete, long successfulEvents, long failedEvents, String message);
+    void onStatusChange(String jobId, int percentComplete, JobStatus jobStatus, long successfulEvents, long failedEvents, String message);
```

Comment:
Please add docs for new parameters and I think it should String jobStatus and not JobStatus jobStatus,